platform/linux-generic/odp_timer.c

```diff
@@ -844,6 +844,7 @@ static int timer_res_init(void)
 
 	sigev.sigev_notify = SIGEV_THREAD_ID;
 	sigev._sigev_un._tid = (pid_t)syscall(SYS_gettid);
+	sigev.sigev_value.sival_ptr = NULL;
```

Comment:
Is this a bug fix? If yes, it should have a Bug created for it with this being the fix patch.